Backend/utils/db_utils.py

```diff
@@ -0,0 +1,224 @@
+
+      attribute_definitions = settings.get("AttributeDefinitions")
+      key_schema = settings.get("KeySchema")
+      provisioned_throughput = settings.get("ProvisionedThroughput")
+      table_name = settings.get("TableName")
+
+      # check for required parameters
+      if table_name is not None and key_schema is not None and attribute_definitions is not None and provisioned_throughput is not None:
+        if len(key_schema) > 0 and len(attribute_definitions) > 0:
+          table = dbInstance.create_table(
+              AttributeDefinitions=attribute_definitions,
+              KeySchema=key_schema,
+              ProvisionedThroughput=provisioned_throughput,
+              TableName=table_name
+            )
+          f.close()
+          return table
+      f.close()
+      raise(IOError)
+  except IOError as e:
+    message = "Unable to create table from file"
+    logger.exception(message + ': ' + settings_file + '\n' + str(e))
+    return message
+
+def createTableFromJson(settings_json, dbInstance):
+  if (type(settings_json) == str):
+    settings = json.loads(settings_json)
+  else:
+    settings = settings_json
+
+  attribute_definitions = settings.get("AttributeDefinitions")
+  key_schema = settings.get("KeySchema")
+  provisioned_throughput = settings.get("ProvisionedThroughput")
+  table_name = settings.get("TableName")
+
+  # check for required parameters
+  if table_name is not None and key_schema is not None and attribute_definitions is not None and provisioned_throughput is not None:
+    if len(key_schema) > 0 and len(attribute_definitions) > 0:
+      table = dbInstance.create_table(
+          AttributeDefinitions=attribute_definitions,
+          KeySchema=key_schema,
+          ProvisionedThroughput=provisioned_throughput,
+          TableName=table_name
+        )
+      return table
+  message = "Unable to create table from json"
+  logger.error(message)
+  return message
+
+def preloadDataFromFile(fileName, table):
+  try:
+    with open(fileName) as json_file:
+      data = json_file.read()
+      items = json.loads(data)
+      for item in items:
+        if "email" in item.keys() and "password" in item.keys() and "activationId" in item.keys() and "active" in item.keys():
+          email = item['email']
+          password = item['password']
+          activationId = item['activationId']
+          active = item['active']
+
+          table.put_item(
+            Item={
+               'email': email,
+               'password': password,
+               'activationId': activationId,
+               'active': active
+            }
+          )
+        else:
+          message = "Unable to add item to table " + table.describe_table + ". Missing email, password, activationId, or active status"
+          logger.error(message)
+  except IOError as e:
+    message = "Unable to load data into table"
+    logger.exception(message + '\nJSON:\n' + jsonData + '\n' + str(e))
+    return message
```

Comment:
This method only sometimes returns something. When it success it returns none. That's kind of counter to what convention is. If we're logging the message here do we really need to return it?